1-exercises/C-highlight-words/script.js

```diff
@@ -1,6 +1,32 @@
-function highlightWords(paragraph, colours) {
-  // Write your code here...
-}
+function highlightWords(paragraph, colors) {
```

Comment:
Please fix the syntax error in this file
![image](https://user-images.githubusercontent.com/54252969/194708250-c12941ec-b88c-4b14-a08c-bbb62c16bb00.png)